lib/customers.js

```diff
@@ -625,6 +625,57 @@ function updateIdCardData (patch, id) {
     })
 }
 
+/**
+ * @param {String} imageData customer t&c photo data
+ * @returns {Promise<Object>} new patch to be applied
+ */
+function updateTxCustomerPhoto (imageData) {
```

Comment:
This function has a lot in common with `updatePhotoCard()`. Any chance we can get a refactor going on here?

Reply:
Hey! I believe we will soon refactor this part of the code to improve folder management, etc.  But I'll ask Rafael